src/controllers/wishlistController.js

```diff
@@ -1,5 +1,34 @@
 const mongoose = require('mongoose');
 const Wishlist = require('../models/Wishlist');
+const { StatusCodes } = require('http-status-codes');
+
+//Create wishlist
+const createWishlist = async (req, res) => {
+  const { albumId } = req.body; //frontend sends the albumId in the request body
```

Comment:
For now, since we're creating an empty wishlist, let's remove this line

Reply:
Done